open-metadata-distribution/open-metadata-assemblies/src/main/assemblies/egeria-omag.xml

```diff
@@ -66,6 +66,8 @@
                 <include>org.odpi.egeria:discovery-service-connectors:*</include>
                 <include>org.odpi.egeria:governance-action-connectors:*</include>
                 <include>org.odpi.egeria:files-integration-connectors:*</include>
+                <include>org.odpi.egeria:kafka-integration-connector:*</include>
+                <include>org.odpi.egeria:openapi-integration-connector:*</include>
```

Comment:
The gradle assembly uses the build.gradle file in this directory to compose an equivilent assembly

new components are added by
a) Adding a dependency
```
 dependencies {

     // Dependencies for the main server assembly - this is modeled on the pom.xml
     // In future we may be able to select by type or label
     implementation project(':open-metadata-implementation:adapters:open-connectors:repository-services-connectors:audit-log-connectors:audit-log-  console-connector')
     ```
b) Adding the particular library ie:
```
            into('server/lib') {
                 // All our connectors, plus discovery sample & the security plugin
                 from { project(':open-metadata-implementation:adapters:open-connectors:repository-services-connectors:audit-log-connectors:audit-  log-console-connector').jar }
                 ```
                 
The gradle assembly is not currently used, but it is built and I have tried to keep it up to date. When we finish the gradle work there will be more checks needed to ensure we have the same content as the maven built version

FYI: @mandy-chessell